### PR TITLE
Base template and auth_app were added

### PR DESCRIPTION
**Pull request by d-karpov**
- Base.html was added to geekhunter app
- auth_app was created for future dev
- login.html and small local style was added to auth_app
- requirements was changed
- settings was updated for current conf

Folks, please review, comments appreciated if any )

### CODE REVIEW

**Review by lobanovjkee — DISMISSED** [review]

File: geekhunter/templates/geekhunter/base.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    {% endblock %}
+
+    {% block js %}
+    {% endblock %}
+</head>
+<body class="{% if request.resolver_match.url_name == 'login' %}bg-dark text-center body-login{% else %}d-flex flex-column h-100{% endif %}">
+{% if request.resolver_match.url_name != 'login' %}
+    {% include './includes/inc_menu.html' %}
+{% endif %}
+<main class="{% if request.resolver_match.url_name == 'login' %}form-signin{% else %}container{% endif %}">
```
Comment (lobanovjkee) [inline geekhunter/templates/geekhunter/base.html:28]: Нужно убрать `{% else %}container`.

Comment (d-karpov) [reply geekhunter/templates/geekhunter/base.html]: Corrected, PR was updated

**Review by lobanovjkee — APPROVED** [review]

File: auth_app/urls.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from auth_app.views import *
```
Comment (KrisMelikova) [inline auth_app/urls.py:2]: Maybe better `from auth_app.views import login`?

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
 asgiref==3.4.1
-Django==3.2
+Django
```
Comment (KrisMelikova) [inline requirements.txt:2]: My proposal is to fix major version (`Django=="4.*"`). What do you think?

**Review by KrisMelikova — COMMENTED** [review]
We can change it later ( or not :))

File: auth_app/urls.py (hunk 2 above)
Comment (d-karpov) [reply auth_app/urls.py]: Correction done.

File: requirements.txt (hunk 3 above)
Comment (d-karpov) [reply requirements.txt]: Done.

**Review by KrisMelikova — APPROVED** [review]

**Review by lobanovjkee — APPROVED** [review]

**Pull request merged**